ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vlan_interfaces.py

```diff
@@ -104,8 +104,12 @@ def _check_virtual_router_mac_address(vlan_interface_config: dict, variables: li
 
         # Only set Anycast v6 GW if VARPv6 is not set
         if vlan_interface_config.get("ip_virtual_router_addresses") is None:
-            vlan_interface_config["ipv6_address_virtual"] = svi.get("ipv6_address_virtual")
-            vlan_interface_config["ipv6_address_virtuals"] = svi.get("ipv6_address_virtuals")
+            if (ipv6_address_virtual := svi.get("ipv6_address_virtual")) is not None:
+                vlan_interface_config["ipv6_address_virtuals"] = [ipv6_address_virtual]
```

Comment:
So here we make a conversion that can be tricky to detect maybe lets add a comment?

Reply:
done

Reply:
done